readme.md

```diff
@@ -1,2 +1,3 @@
 README
-plz
+
```

Comment:
clarity maximized 👍

Reply:
this is perfect

Reply:
hey @botany-lead how are you?

Reply:
hey @botany-lead can you check this out?
cc @botany-demo @botany-dummy

Reply:
this is really helpful

Reply:
i couldn't agree more